Packs/EWS/Integrations/EWSO365/EWSO365.yml

```diff
@@ -106,39 +103,37 @@ configuration:
   required: false
   type: 4
   hidden: true
-description: The new EWS O365 integration uses OAuth 2.0 protocol and can be used
-  with Exchange Online and Office 365 (mail).
+- defaultvalue: '1'
+  display: Incidents Fetch Interval
+  name: incidentFetchInterval
+  required: false
+  type: 19
+- additionalinfo: Default is to filter by received-time, which works well if the folder is an "Inbox" - but for a folder emails are dragged into for attention, if we filter by received-time, out-of-order processing of emails means some are ignored. Filtering by modified-time works better for such a scenario.
```

Comment:
Consider adding the explanation about the modification before dragging to folder.